test/integration/features/coolstore.feature

```diff
@@ -6,14 +6,17 @@ Feature: Cool Store
   After which I will import image streams
   Finally I will deploy the Cool Store
   
-  Scenario: User enables the 'xpaas' add-on and creates a Minishift instance
+  @minishift-only
```

Comment:
Tag again is not truly needed. However +1 for splitting up the scenario.

Reply:
tag removed

Reply:
IMO this is not the right pr for splitting the scenario. Will create an issue unless there is a hard requirement of splitting. As of now let the original file be as it is.